modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/KnowledgeBaseArticleResourceTest.java

```diff
@@ -75,6 +80,54 @@ public void testDeleteKnowledgeBaseArticleMyRating() throws Exception {
 					irrelevantKnowledgeBaseArticle.getId()));
 	}
 
+	@Test
+	public void testPatchKnowledgeBaseArticleDisplayDate() throws Exception {
```

Comment:
Instead of saving the original date, posting a modified version and then later patching back the original date again, why don't you do the opposite? As it is right now the code looks a bit confusing.
```
var randomArticle = randomArticle();

var postedArticle = post(randomArticle);

assertEquals(randomArticle.date, postedArticle.date); // check that posting saves expected value.

postedArticle.date = addHours(postedArticle.date, randomPositiveInteger);

var patchedArticle = patch(postedArticle);

assertEquals(patchedArticle.date, postedArticle.date); // check that patching updates the value as expected.
```

Reply:
You may even skip the first asserting (the one for post), as that is already checked in the test for post.